src/dune_util/persistent.ml

```diff
@@ -72,7 +72,36 @@ module Make (D : Desc) = struct
                 None
               | d -> Some d
             else None)
-    else None
+    in
+    let read_file =
+      lazy
+        (match Dune_stats.global () with
+        | None -> read_file
+        | Some stats ->
+          fun file ->
+            let start = Unix.gettimeofday () in
+            let res = Result.try_with (fun () -> read_file file) in
+            let event =
+              let stop = Unix.gettimeofday () in
+              let module Event = Chrome_trace.Event in
+              let module Timestamp = Event.Timestamp in
+              let dur = Timestamp.of_float_seconds (stop -. start) in
+              let common =
+                Event.common_fields ~name:"Persistent"
```

Comment:
Are users going to see only "Persistent" in the trace? Seems too terse/hard to guess what it is. How about "Loading persistent Dune state"?

Reply:
I changed it to `Loading Persistent State`. `Dune` seems obvious given the context.

Reply:
I'm not so sure actually. There can be millions of events in the trace and which of them correspond to Dune may not be obvious. I don't see a good reason to be terse here.

Reply:
Do you mean if we allow events to be written by other programs? Right now only dune is allowed to write to this trace file.

Reply:
I mean, Dune runs lots of processes which can also be loading all sorts of states.

Reply:
Yup. But none of them can record anything in the trace file. Although we would like to change that https://github.com/ocaml/dune/issues/3449

Anyway, I can add "dune" to the name. I'm just not sure what you mean that it can be confused for some other state.

Reply:
Thanks! 

> But none of them can record anything in the trace file.

They currently can't but Dune does that for them, right? Dune will create lots of events about other processes and distinguishing those from internal events may be non-trivial.